pkg/models/folders.go

```diff
@@ -79,6 +79,10 @@ type CreateFolderCommand struct {
 	Result *Folder `json:"-"`
 }
 
+type MoveFolderCommand struct {
+	ParentUID *string `json:"parentUID"`
```

Comment:
As much as I like this Go style, in the API we tend to use `Uid` and `Id` suffices for JSONs